ios/brave-ios/Sources/BraveWallet/Crypto/Stores/TransactionConfirmationStore.swift

```diff
@@ -573,6 +573,10 @@ public class TransactionConfirmationStore: ObservableObject, WalletObserverStore
       value = details.fromAmount
       fiat = details.fromFiat ?? ""
 
+      if activeParsedTransaction.transaction.txType == .solanaCompressedNftTransfer {
+        symbol = "SOL"
```

Comment:
We might want a comment linking to https://github.com/brave/brave-browser/issues/40353 or iOS frontend issue for this to update later so we can show correct info here.